src/workerd/server/tests/python/BUILD.bazel

```diff
@@ -1,59 +1,67 @@
 load("//:build/wd_test.bzl", "wd_test")
 
+load("@bazel_skylib//rules:copy_file.bzl", "copy_file")
+
+copy_file(
+    name = "pyodide-dev.capnp.bin@rule",
+    src = "//src/pyodide:pyodide.capnp.bin",
+    out = "pyodide-bundle-cache/pyodide-dev.capnp.bin"
+)
+
 wd_test(
     src = "hello/hello.wd-test",
-    args = ["--experimental"],
+    args = ["--experimental", "--pyodide-bundle-disk-cache-dir", "$(location pyodide-dev.capnp.bin@rule)/.."],
     data = glob(
         [
             "hello/*",
         ],
         exclude = ["**/*.wd-test"],
-    ),
+    ) + ["pyodide-dev.capnp.bin@rule"],
 )
 
 wd_test(
     src = "env-param/env.wd-test",
-    args = ["--experimental"],
+    args = ["--experimental", "--pyodide-bundle-disk-cache-dir", "$(location pyodide-dev.capnp.bin@rule)/.."],
     data = glob(
         [
             "env-param/*",
         ],
         exclude = ["**/*.wd-test"],
-    ),
+    ) + ["pyodide-dev.capnp.bin@rule"],
 )
 
 wd_test(
     src = "random/random.wd-test",
-    args = ["--experimental"],
+    args = ["--experimental", "--pyodide-bundle-disk-cache-dir", "$(location pyodide-dev.capnp.bin@rule)/.."],
     data = glob(
         [
             "random/*",
         ],
         exclude = ["**/*.wd-test"],
-    ),
+    ) + ["pyodide-dev.capnp.bin@rule"],
 )
 
 # Disabled because it tests the same thing as the import test defined in import_tests.bzl
 # wd_test(
 #     src = "langchain/langchain.wd-test",
-#     args = ["--experimental", "--disk-cache-dir", "../all_pyodide_wheels"],
+#     args = ["--experimental", "--pyodide-package-disk-cache-dir", "../all_pyodide_wheels", "--pyodide-bundle-disk-cache-dir", "$(location pyodide-dev.capnp.bin@rule)/.."],
 #     data = glob(
 #         [
 #             "langchain/*",
 #         ],
 #         exclude = ["**/*.wd-test"],
-#     ) + ["@all_pyodide_wheels//:whls"],
+#     ) + ["@all_pyodide_wheels//:whls", "pyodide-dev.capnp.bin@rule"],
 # )
```

Comment:
Should we remove this?

Reply:
It's removed in the separate "refactor python bazel tests" PR that hood requested